src/test/kotlin/build/buf/intellij/fixes/BufFixesTest.kt

```diff
@@ -111,7 +108,6 @@ class BufFixesTest : BufTestBase() {
     }
 
     fun testIgnoreV2ModulesDisabledTrue() {
-        configureCLI(System.getProperty("BUF_BETA_CLI"))
```

Comment:
Is the `configureCLI` function still needed or can it be deleted, too?

Was this function what previously populated the testData/cache folders? Was there a particular reason to have the cache files checked in instead of populated at test time?

Reply:
I'd like to keep it around in case we need to test with different versions of the CLI in the future for any reason.